DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: sotkanet
 Type: Package
 Title: Sotkanet Open Data Access and Analysis
-Version: 0.9.79
-Date: 2022-02-01
+Version: 0.10.0
```

Comment:
Numbering must start from 1 -> make this 0.10.1

(Or should we already go for official release 1.1.1?